sdk/pinocchio/src/sysvars/tests/slot_hashes_tests.rs

```diff
@@ -0,0 +1,855 @@
+#[cfg(test)]
+use super::*;
+mod tests {
```

Comment:
Considering the whole module is called `slot_hashes_tests` is there a reason to introduce yet another nested submodule?
Why not put everything directly at the top level?